e2e-tests/cypress/support/selectorsManageIncidents.ts

```diff
@@ -63,15 +63,15 @@ export const MANAGE_SIGNALS = {
   buttonCloseMijnFilters: '[data-testid=closeBtn]',
   buttonFilter: '[data-testid=filterModalBtn]',
   clearSearchTerm: '[aria-label="Close"]',
-  firstSignalAdres: 'tbody > :nth-child(1) > :nth-child(9) > span > a',
-  firstSignalDag: '[data-testid="incidentDaysOpen"] > span > a',
-  firstSignalDatumTijd: 'tbody > :nth-child(1) > :nth-child(5) > span > a',
-  firstSignalIcon: 'tbody > :nth-child(1) > :nth-child(1) > span > a > span',
-  firstSignalId: 'tbody > :nth-child(1) > :nth-child(3) > span > a',
-  firstSignalStadsdeelName: 'tbody > :nth-child(1) > :nth-child(8) > span > a',
-  firstSignalStatus: 'tbody > :nth-child(1) > :nth-child(8) > span > a',
-  firstSignalSubcategorie: 'tbody > :nth-child(1) > :nth-child(6) > span > a',
-  firstSignalUrgentie: 'tbody > :nth-child(1) > :nth-child(1) > span > a',
+  firstSignalAdres: 'tbody > :nth-child(1) > :nth-child(9) > a',
+  firstSignalDag: '[data-testid="incidentDaysOpen"] > a',
+  firstSignalDatumTijd: 'tbody > :nth-child(1) > :nth-child(5) > a',
+  firstSignalIcon: 'tbody > :nth-child(1) > :nth-child(1) > a > span',
+  firstSignalId: 'tbody > :nth-child(1) > :nth-child(3) > a',
+  firstSignalStadsdeelName: 'tbody > :nth-child(1) > :nth-child(8) > a',
+  firstSignalStatus: 'tbody > :nth-child(1) > :nth-child(8) > a',
+  firstSignalSubcategorie: 'tbody > :nth-child(1) > :nth-child(6) > a',
+  firstSignalUrgentie: 'tbody > :nth-child(1) > :nth-child(1) > > a',
```

Comment:
Still pretty fragile; would be better to reference the individual cells by `data-testid` attribute.

Reply:
Agreed - changed it to data-testid's